ai-platform-unified/notebooks/unofficial/featurestore/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1342 @@
+{
```

Comment:
<p>Can this be done programmatically? This won't be able to pass CI testing in its current state.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Do we have a BigQuery NB example? I can try to look for a programmatic way from there.</p>

Reply:
<p>changed to using bq clients</p>